server.go

```diff
@@ -166,8 +175,48 @@ func NewFromConfig(conf Config) (ret Server, err error) {
 	ret.HTTPAddr = conf.HTTPAddress
 	ret.ForwardAddr = conf.ForwardAddress
 
+	if conf.TcpAddress != "" {
+		ret.TCPAddr, err = net.ResolveTCPAddr("tcp", conf.TcpAddress)
+		if err != nil {
+			return
```

Comment:
This function seems to be used "named returns". I followed the pattern that already exists here. For example, see how errors are handled when resolving the UDP address, directly above here on line 171